apps/dashboard/lib/ratelimitValues.ts

```diff
@@ -0,0 +1,10 @@
+// Values for route types
+
+export const CREATE_LIMIT = 5;
+export const CREATE_LIMIT_DURATION = 3;
```

Comment:
What's the unit here?
we use milliseconds almost everywhere and that should be default unless clearly labeled